pytest_bdd/generation.py

```diff
@@ -47,7 +47,11 @@ def generate_code(features, scenarios, steps):
     grouped_steps = group_steps(steps)
     template = template_lookup.get_template("test.py.mak")
     return template.render(
-        features=features, scenarios=scenarios, steps=grouped_steps, make_python_name=make_python_name
+        features=features,
```

Comment:
This library now uses [black](https://github.com/psf/black) as the code formatter.
Make sure you run `black` against the committed files (I use `pre-commit` hooks to automatically detect if that was not done, more info [here](https://github.com/psf/black#version-control-integration))

Reply:
Done. It also reformatted another file which I've committed separately.